pkg/operator/clustermemberremovalcontroller/clustermemberremovalcontroller_test.go

```diff
@@ -25,6 +27,28 @@ import (
 	"k8s.io/client-go/tools/cache"
 )
 
+var (
+	bootstrapComplete = &corev1.ConfigMap{
```

Comment:
I think we have those in test utils already: https://github.com/openshift/cluster-etcd-operator/blob/master/pkg/testutils/testutils.go#L168-L181

Reply:
also saw it's duplicated in `bootstrap_teardown_controller_test.go`, if you want to refactor the extra mile :)

Reply:
lets do it 👍🏽

Reply:
done